content/blog/high-utilization-in-systems/index.md

```diff
@@ -0,0 +1,114 @@
+
+While every team will face distinct challenges, most of them will face similar consequences.
+
+**Communication**
+
+One of the first things that breaks down is communication. Asynchronous messages start taking longer to be answered or not answered at all in public and private channels. Emails or issues will tend to be forgotten until someone volunteers or gets "voluntold". You will notice less engagement in meetings, with people multitasking or not participating at all. This in turn decreases the amount of collaboration and trust among team members.
+
+**Shared responsibilities**
+
+With every team member under high utilization shared responsibilities starts falling through the cracks. These are the responsibilities that are not tied to any single individual, but that are shared among the team. Among those we can list:
+
+* Mentorship
+* Knowledge sharing
+* Code reviews
+* Documentation
+
+Among those, code reviews and mentorship directly impacts the cycle time as well.
+
+**Improvement and Innovation**
+
+It is challenging to think strategically when you don't have time. Teams under high utilization tend to stagnate, keeping their current processes in place for far longer than they should. This happens not because they are unable to improve, but simply because they don't have the time to consider how things could be improved or innovated.
+
+Notice how all of those consequences are critical for an organization to remain healthy in the long-term. It's not a coincidence that they are deprioritized.
+
+## Why does this myth of high utilization persist?
+
+If doing things this way impacts delivery speed and quality in the long run, why do we still see high utilization as a positive signal? My theory is that most of our industry is caught up playing short term games. 
+
+Leadership is expected to provide results in short-term cycles, hence management gets recognized for delivering projects in tight timelines and developers are rewarded when they are able to ship tasks as fast as possible. None of these behaviours are responsible for creating better products, teams or culture in the long run. This reminds me of a quote from Dave Packard:
+
+> “More Organisations Die Of Indigestion Than Starvation”
+
+This quote really resonates with me. We need teams focused on a small number of objectives that are aligned with the company vision. High utilization doesn't allow for strategic planning, getting teams caught up in a tactical mindset.
+
+## Things to look out for
+
+If you are not sure whether your team is under high utilization or not (you probably know) you can look for the following characteristics:
+
+* Team members working alone or in small teams due too many initiatives
+* Poor engagement in meetings
+* Asynchronous channels (Slack, email, etc) drying up in participation or being slow/ignored
+* Pull request lead cycle taking a long time
+    - PRs are taking longer to be opened (high complexity/lower quality in the codebase)
```

Comment:
not just PRs taking longer to be opened, but projects as a whole slow down

Reply:
Often times the first 75% is shipped fast, but the last 25% gets heavily de-prioritized as other tasks come in. This is what also leads to very drawn out projects which seem like they're almost done, but the work drags out.

Reply:
This is concerning, why would 25% of the project be deprioritized if the project itself is a priority?